packages/core-sdk/src/types/common.ts

```diff
@@ -67,3 +67,21 @@ export type ValidatedLicensingConfig = {
  * Will be converted to bigint for contract calls.
  */
 export type TokenAmountInput = bigint | number;
+
+/**
+ * The type of revenue share.
+ */
+export enum RevShareType {
+  /**
+   * The commercial revenue share.
```

Comment:
think these docs are not needed. it doesn't have any new info that the var does of `COMMERCIAL_REVENUE_SHARE` does not tell you already.